packages/docs/src/components/PaddingSpacingExample.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const spacerOpts = ['0', '05', '1', '2', '3', '4', '5', '6', '7'];
```

Comment:
This is way out of scope and not worth doing in this first clone version of the doc site, but we could theoretically import these from our tokens package in the future 🤔